### DISCUSSION TITLE
Bot message if new thread in forum channel

### DISCUSSION THREAD

**Discussion by ArtyomCZ** [discordjs/discord.js#9137] [Q&A]
Hi,

I was trying to create a code so that the bot would send a message to another text channel when a new thread was created in the forum channel. But its not working.

``` 
client.on('threadCreate', async (thread) => {
    if (thread.parent.id === 'ID1') { 
      const channel = client.channels.cache.get('ID2');
      await channel.send('Message for text channel');
    }
  });
```
Is there any way to do it?

**Comment by quentinchap** [comment]
Did you try thread.parentId === 'ID1' ?  Here ID1 is the id of the forum channel not the post itself. 
To finish what error do you catch?

**Reply by ArtyomCZ** [reply]
thread.parentId === 'ID1' is id of forum channel. Id2 is channel of text channel where bot should send message if new thread in forum channel created.

Getting error parent.thread is probably null.

![image](https://user-images.githubusercontent.com/16862369/220904508-1ff3aee7-f3c6-4c1d-a4d6-47990b9a4d4f.png)
